doc/build/orm/declarative_tables.rst

```diff
@@ -1158,6 +1158,10 @@ additional columns are present on mapped subclasses that have
 no :class:`.Table` of their own.  This is illustrated in the section
 :ref:`single_inheritance`.
 
+Note that in a similar fashion a :func:`_orm.relationship` can be added
```

Comment:
why dont we use ``.. seealso::`` to make a block for this


.. seealso::

   :ref:`orm_declarative_table_adding_relationship` - similar examples for :func:`_orm.relationship`

Reply:
will do this in gerrit

Reply:
Oh ok, no need for me to tinker then?

Reply:
nope I had it checked out to check if sphinx liked it etc, so it was just faster to apply the change.

thanks for starting the PR